src/lib/FlightTasks/tasks/Auto/FlightTaskAuto.hpp

```diff
@@ -43,9 +43,9 @@
 #include <uORB/topics/position_setpoint_triplet.h>
 #include <uORB/topics/position_setpoint.h>
 #include <uORB/topics/home_position.h>
-#include <uORB/topics/position_controller_status.h>
 #include <uORB/topics/vehicle_status.h>
 #include <lib/ecl/geo/geo.h>
+#include "lib/FlightTasks/tasks/Utility/ObstacleAvoidance.hpp"
```

Comment:
This include is not required. You need to add the the library `FlightTaskUtility` to `CMakerLists.txt` within the FlightaskAuto-folder:
```
target_link_libraries(FlightTaskAuto PUBLIC FlightTask FlightTaskUtility)
```
If you do that, then you can include the avoidance library without relative path:
```
#include "ObstacleAvoidance.hpp"
```